sourcemash/api/users.py

```diff
@@ -50,12 +49,14 @@ def __init__(self):
     def get(self):
         return { 'user': marshal(current_user, user_fields) }
```

Comment:
So what happens if your login is incorrect for loginOrShowErrors or registerOrShowErrors?